src/index.js

```diff
@@ -1,2 +1,5 @@
 export { PLS } from './pls';
 export { KOPLS } from './kopls';
+export { OPLS } from './OPLS';
+export { oplsNIPALS } from './oplsNIPALS';
+export { summaryMetadata } from './summaryMetadata';
```

Comment:
I'm not sure this function makes sense in this package. It doesn't seem specific to PLS and isn't necessary to execute the algorithms